mypy.ini

```diff
@@ -99,4 +99,10 @@ ignore_missing_imports = True
 ignore_missing_imports = True
 
 [mypy-ipywidgets.*]
+ignore_missing_imports = True
+
+[mypy-hypothesis.*]
+ignore_missing_imports = True
+
+[mypy-hypothesis_jsonschema.*]
```

Comment:
If dont ignore, missing imports, will get error in mypy and github test crashes?